lightning/src/ln/functional_test_utils.rs

```diff
@@ -1190,6 +1190,28 @@ macro_rules! get_route_and_payment_hash {
 	}}
 }
 
+#[cfg(test)]
+#[macro_export]
+macro_rules! fail_to_get_route {
```

Comment:
We're deleting this, right? You could also take the `get_route` commit from #1451 as a replacement.

Reply:
>I'm not sure why it didn't work? $expr is generic across any express, as long as all the call sites use the same type it should always work. What did you try and what error did you get?

Yeah, the problem is exactly that `expr` is too generic because contains is only on the string type, strange theory type :/. However, I will commit your suggestion, and in case we revert it

>We're deleting this, right? You could also take the get_route commit from https://github.com/lightningdevkit/rust-lightning/pull/1451 as a replacement.

Right! I had this on my todo list for review, I think yes, having a `get_route` will be helpful, thank you!